### ISSUE TITLE
Update flutter plugin to flutter 3

### ISSUE THREAD

**Issue by selvasoft** [Netvent/storyly-mobile#189]
We are trying to move our app to flutter 3. But storyly plugin is not updated to flutter 3. So, its a blocker for us to move to flutter 3.
So, storyly plugin needs to be updated to flutter 3.

## Environment Details
**Storyly SDK Version:** 1.20.0
**Flutter SDK Version:** 3.0.1
**Platform:**
- [x] Android

**Device Model:** Redmi note 9 pro
**Device OS Version:** Android 11
**Additional Info:** This is not a device specific issue.

## Expected Behavior
App should compile properly.

## Current Behavior
App is not compiling and the following error is thrown.
```e: /Users/selva.kumar/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/storyly_flutter-1.20.0/android/src/main/kotlin/com/appsamurai/storyly/storyly_flutter/FlutterStorylyView.kt: (22, 1): Class 'FlutterStorylyViewFactory' is not abstract and does not implement abstract base class member public abstract fun create(p0: Context?, p1: Int, p2: Any?): PlatformView defined in io.flutter.plugin.platform.PlatformViewFactory
e: /Users/selva.kumar/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/storyly_flutter-1.20.0/android/src/main/kotlin/com/appsamurai/storyly/storyly_flutter/FlutterStorylyView.kt: (25, 5): 'create' overrides nothing

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':storyly_flutter:compileDebugKotlin'.
```

## Steps to Reproduce
1. Clone the repo.
2. Use flutter 3.0.1
3. Run the example
4. You will see the error.

**Comment by BrunoJurkovic** [comment]
@levent-oral Can we please get this checked out? It's causing some big issues at our end and we might have to downgrade if you don't fix this issue soon.

**Comment by levent-oral** [comment]
Duplicate of [#184](https://github.com/Netvent/storyly-mobile/issues/184)

We've released [storyly_flutter@1.23.0](https://github.com/Netvent/storyly-mobile/blob/master/flutter/storyly_flutter/CHANGELOG.md#1230-27062022) version to support Flutter 3.0.
